src/app/services/storage.service.ts

```diff
@@ -18,9 +20,11 @@ export class StorageService {
   uploadFile(file: File) {
     const storageRef = this.firebaseStorage.storage.ref();
 
-    storageRef.child(this.user.uid).child(Date.now() + '.' + file.name.split('.').pop()).put(file)
+    storageRef.child(this.user.uid)
+      .child(Date.now() + '.' + file.name.split('.').pop())
+      .put(file)
       .then(snapshot => {
-        console.log('Uploaded a file!');
+        this.isUploadedSource.next(snapshot.state);
```

Comment:
do we need the observable seeing as this returns a promise?

Reply:
I agree. Fixed it as well.